Remotion/Data/DomainObjects/Persistence/StorageProviderManager.cs

```diff
@@ -89,20 +90,25 @@ public StorageProvider GetMandatory (StorageProviderDefinition providerDefinitio
     }
 #endif
 
-    if (_storageProviders.Contains(providerDefinition.Name))
-      return _storageProviders[providerDefinition.Name]!;
+    if (_storageProviders.TryGetValue(providerDefinition.Name, out var storageProvider))
+      return storageProvider!;
```

Comment:
`storageProvider` cannot be `null`, because we have a `Dictionary<string, IReadOnlyStorageProvider>`, where the value type is non-nullable, so the `!` does not do anything.